app/views/assignments/show.html.erb

```diff
@@ -128,11 +128,7 @@
         <%# Since the student has a group already, we display information about the group only %>
         <%# Check if groupname should be displayed %>
```

Comment:
You can delete this comment as well, since it no longer applies

Reply:
Done!